network-api/networkapi/wagtailpages/templates/wagtailpages/research_detail_page.html

```diff
@@ -2,15 +2,15 @@
 {% load i18n wagtailcore_tags wagtailimages_tags %}
 
 {% block research_hub_content %}
-  <div class="tw-grid tw-grid-cols-1 medium:tw-grid-cols-[12rem_1fr_14rem] large:tw-grid-cols-[14rem_1fr_16rem] xlarge:tw-grid-cols-[14rem_1fr_16rem] tw-gap-5">
+  <div class="tw-grid tw-grid-cols-1 tw-grid-rows-[auto_auto_1fr] medium:tw-grid-cols-[12rem_1fr_14rem] large:tw-grid-cols-[14rem_1fr_16rem] xlarge:tw-grid-cols-[14rem_1fr_16rem] tw-gap-5">
 
     <div id="breadcrumbs-container" class="medium:tw-col-start-2 medium:tw-col-end-4">
       {% include "wagtailpages/fragments/research_breadcrumb.html" with breadcrumb_list=breadcrumbs %}
     </div>
 
     <div id="title-and-meta" class="medium:tw-col-start-2 medium:tw-col-end-4 xlarge:tw-col-end-3 tw-min-w-0">
       <h1>{{ page.title }}</h1>
-      <div class="tw-flex tw-flex-row tw-text-xs large:tw-text-base tw-text-gray-80 tw-mb-2 tw-uppercase tw-divide-x tw-divide-gray-20">
+      <div class="tw-flex tw-flex-row tw-text-xs large:tw-text-base tw-text-gray-80 tw-uppercase tw-divide-x tw-divide-gray-20 tw-mt-4">
```

Comment:
**Suggestion** Since the H1 is already setting the margin bottom to this same size the `tw-mt-4` has no effect. Therefore I think it we can remove this.

Reply:
Thanks Tibor! Updated